Gordon360/Web.config

```diff
@@ -116,5 +116,5 @@
+  <connectionStrings><add name="StudentTimesheetsEntities" connectionString="metadata=res://*/Models.StudentTimesheet_DB_Models.csdl|res://*/Models.StudentTimesheet_DB_Models.ssdl|res://*/Models.StudentTimesheet_DB_Models.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=admintrainsql.gordon.edu;initial catalog=StudentTimesheets;integrated security=True;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" /><add name="MyGordonEntities" connectionString="metadata=res://*/Models.MyGordon_DB_Models.csdl|res://*/Models.MyGordon_DB_Models.ssdl|res://*/Models.MyGordon_DB_Models.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=admintrainsql.gordon.edu;initial catalog=MyGordon;integrated security=True;multipleactiveresultsets=True;application name=EntityFramework&quot;" providerName="System.Data.EntityClient" /><add name="StaffTimesheetsEntities" connectionString="metadata=res://*/Models.StaffTimesheets_DB_Models.csdl|res://*/Models.StaffTimesheets_DB_Models.ssdl|res://*/Models.StaffTimesheets_DB_Models.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=admintrainsql.gordon.edu;initial catalog=StaffTimesheets;integrated security=True;multipleactiveresultsets=True;application name=EntityFramework&quot;" providerName="System.Data.EntityClient" /><add name="CCTEntities1" connectionString="metadata=res://*/Models.CCT_DB_Models.csdl|res://*/Models.CCT_DB_Models.ssdl|res://*/Models.CCT_DB_Models.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=admintrainsql.gordon.edu;initial catalog=CCT;integrated security=True;multipleactiveresultsets=True;application name=EntityFramework&quot;" providerName="System.Data.EntityClient" /></connectionStrings>
```

Comment:
There's a slight difference at the very end, but it means the same thing as before, it seems.